miopen_dialect_idea.md

```diff
@@ -380,6 +380,181 @@ miopen.gridwise_gemm(%filter_gemmK_gemmM, %output_gemmK_gemmN, %input_gemmM_gemm
 
 -------------------------------------------------------------------------------
 
+conv backward weight transform
+===============
+
+an example based on nchw/kcyx/nkhw:
+
+```mlir
+// filter tensor
+%filter_gemmM_gemmN = miopen.transform(%filter) {
+  gridwise_gemm_argument_position = 2
+  layout = [
+    {
+      dimensions = [0],
+      names = ["gemmM"],
+      transformation = "PassThrough",
+      source_dimensions = [0],
+      source_names = ["k"]
+    },
+    {
+      dimensions = [1],
+      names = ["gemmN"],
+      transformation = "Unfold",
```

Comment:
Just to clarify, "Unfold" only applies to KCYX layout, but not, for example CKYX? (which would apply for "Merge")

Reply:
unfold should only apply to consecutive source dimensions.